accession_register.py

```diff
@@ -19,14 +19,14 @@ def parse_args(args_):
         ' Written by Kieran O\'Leary.'
     )
     parser.add_argument(
-        '-register', help='Path to helper accessions register'
+        '-sorted_csv', help='Path to sorted helper CSV'
```

Comment:
It would still be useful to include a reference to this being the accessions register in the help file? Not in the argument, just the help bit.

Reply:
what I've been using with the -register argument is the sorted_csv file that accession.py/batchacession.py creates and puts in the ifiscripts logs folder. it's the one that matches the oe numbers with the accession numbers (you know all of this -sorry). 
i'm naming it sorted_csv in the argument so that it reflects the name of the correct csv file to bring in.
In the help bit: - i didn't call it helper accession register, because then it might get confused with the csv file that this script creates - the actual helper accession register document that we use to fill in the digital accession register. the sorted_csv file, that's inputted into this argument is sort of the helper to the helper.
do you know what I mean? I might not be making much sense, but can explain it in person.

R

Reply:
Using the argument to reflect the filename makes sense, i completely forgot about that.
And i know exactly what you mean regarding the help - thanks for clarifying, all looks good!

Reply:
i've updated line 62 to correct the error. 
should be good to merge now.
let me know
cheers